src/examples/radiuss_tutorial/lesson_04/device_spatial_indexes.cpp

```diff
@@ -0,0 +1,582 @@
+// Copyright (c) 2017-2023, Lawrence Livermore National Security, LLC and
+// other Axom Project Developers. See the top-level LICENSE file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+
+//-----------------------------------------------------------------------------
+///
+/// file: device_self_intersections.cpp
+///
+/// This example ports the naive self-intersection algorithm to additional
+/// execution and memory spaces w/ the help of RAJA and Umpire
```

Comment:
Check this description since there was a copy/paste error above.

Reply:
Thanks! I forgot to update it!

Reply:
Done!